src/test/java/com/googleinterns/zoomtube/servlets/TranscriptServletTest.java

```diff
@@ -0,0 +1,255 @@
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"1.3\",\"duration\":"
+      + "\"3.1\",\"content\":\"All right, so here we are\\nin front of the "
+      + "elephants,\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":4},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"12.7\",\"duration\":\"4.3\",\"content\":"
+      + "\"and that&#39;s, that&#39;s cool.\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":5},"
+      + "\"lectureKey\":{\"kind\":\"lecture\",\"id\":123},\"start\":\"17\",\"duration\":\"1.767\",\""
+      + "content\":\"And that&#39;s pretty much all there is to say.\"},{\"transcriptKey\":{\"kind\""
+      + ":\"TranscriptLine\",\"id\":2},\"lectureKey\":{\"kind\":\"lecture\",\"id\":123},\"start\":"
+      + "\"4.4\",\"duration\":\"4.766\",\"content\":\"the cool thing about these guys\\nis that "
+      + "they have really,\"},{\"transcriptKey\":{\"kind\":\"TranscriptLine\",\"id\":3},\"lectureKey\":"
+      + "{\"kind\":\"lecture\",\"id\":123},\"start\":\"9.166\",\"duration\":\"3.534\","
+      + "\"content\":\"really, really long trunks,\"}]";
+
+  @Before
+  public void setUp() throws ServletException, IOException {
+    localServiceHelper.setUp();
+    servlet = new TranscriptServlet();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    servlet.init(datastore);
+    gson = new GsonBuilder().registerTypeAdapterFactory(GenerateTypeAdapter.FACTORY).create();
+    content = new StringWriter();
+    PrintWriter writer = new PrintWriter(content);
+    when(response.getWriter()).thenReturn(writer);
+  }
+
+  @After
+  public void tearDown() {
+    localServiceHelper.tearDown();
+  }
+
+  @Test
+  public void doGet_getDataInDatastoreForShortVideo() throws ServletException, IOException {
+    putJsonInDatastore(SHORT_VIDEO_JSON, LECTURE_ID_A);
+    when(request.getParameter(TranscriptServlet.PARAM_LECTURE_ID)).thenReturn(LECTURE_ID_A);
+
+    servlet.doGet(request, response);
+    String actualJson = content.toString();
+    List<TranscriptLine> expectedArrayList = extractJsonAsArrayList(SHORT_VIDEO_JSON);
+    List<TranscriptLine> actualJsonArray = extractJsonAsArrayList(actualJson);
+
+    assertThat(actualJsonArray.size()).isEqualTo(expectedArrayList.size());
+  }
+
+  @Test
+  public void doPost_persistDataInDatastoreForShortVideo() throws ServletException, IOException {
+    when(request.getParameter(TranscriptServlet.PARAM_VIDEO_ID)).thenReturn(SHORT_VIDEO_ID);
+    when(request.getParameter(TranscriptServlet.PARAM_LECTURE_ID)).thenReturn(LECTURE_ID_B);
+
+    servlet.doPost(request, response);
+    int actualQueryCount = countEntitiesInDatastore(LECTURE_ID_B);
+    int expectedQueryCount = (extractJsonAsArrayList(SHORT_VIDEO_JSON)).size();
+
+    assertThat(actualQueryCount).isEqualTo(expectedQueryCount);
+  }
+
+  @Test
+  public void doGet_doPost_StoreAndRetrieveShortVideo() throws ServletException, IOException {
```

Comment:
```suggestion
  public void doGet_doPost_storeAndRetrieveShortVideo() throws ServletException, IOException {
```

Reply:
Thanks for catching that! I fixed the capitalization for the other test methods as well.